content/docs/04-clusters/06-cluster-management/15-node-pool.md

```diff
@@ -74,16 +68,35 @@ You can create a new node pool for an active cluster. To create a new node pool
 | **Availability Zones** | The Availability Zones from which to select available servers for deployment. If you select multiple zones, Palette will deploy servers evenly across them as long as sufficient servers are available to do so. |
 | **Disk Size** | Give the required storage size. |
  
-**Note**: Currently Palette does not support Autoscaler for Azure clusters.
 
 <br />
 
-7. Click on **Confirm** to create the new node pool.
+
+<WarningBox>
+
+Some features may not be available for all infrastructure providers. Review each infrastructure provider's node pool configuration settings to learn more.
+
+</WarningBox>
+
+<br />
 
 
-# Change a Node Pool
+## Create a New Node Pool
 
-Palette allows its users to apply changes to a node pool, including its taints label for a running cluster. To make changes to the an active cluster's node pools, follow the steps below. 
+
+### Prerequisites
+
+
+* A Palette deployed cluster.
+
+
+* Sufficient permissions to edit the cluster.
+
+
+### Create Node Pool
+
+
```

Comment:
Should we add the "Log in to Palette" step as the first step and navigate the user in subsequent steps to the point they click on the Nodes tab?

Reply:
That's odd, I could swear I had that as the first step. Let me fix that.